src/term/check/unbounds.rs

```diff
@@ -0,0 +1,9 @@
+use crate::term::Book;
+
+impl Book {
```

Comment:
Unless we plan to do something else with this, I don't think this is a good idea.
This is simple enough that it seems better to me to just always have it inlined